spec/components/services/session.spec.ts

```
@@ -33,6 +33,34 @@ describe("Session", function() {
     });
   });
 
+  describe("find", function() {
+    describe("with existing hash", function() {
+      beforeEach(async function(this: CurrentThisContext) {
+        await this.session.set("my-key", "my-value");
+        await this.session.set("not-the-key", "not-the-value");
+      });
+
+      it("returns array with my-value", async function(this: CurrentThisContext) {
+        expect(await this.session.find("my-")).toEqual({ "my-key": "my-value" });
+      });
+
+      it("returns array with my-value and not-the-value", async function(this: CurrentThisContext) {
```

add a `describe("with empty parameter given")` block here. 

Wouldn't it be better to throw an exception if the given parameter is empty? Especially if we expose the `keys` api as suggested below.